_source/_posts/2024-05-01-okta-rsa.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: blog_post
+title: "Okta at RSA Conference 2024"
+author: [michael-palermo, semona-igama, edunham]
+by: advocate
+communities: [devops,security,mobile,.net,java,javascript,go,php,python,ruby]
+description: "Join Okta at RSA Conference on May 7, 2024"
+tags: []
+tweets:
+- ""
+- ""
+- ""
+image: 
+type: awareness
+---
+
+We are excited to have a presence at RSA Conference 2024! On Tuesday, May 7th, we will speak on [Identity: Your key to stop breaches](https://www.rsaconference.com/usa/agenda/session/Okta%20Identity%20Your%20key%20to%20stop%20breaches). Consider why you don't want to miss that session: 
+
+> Attackers are not breaking in – they are logging in. Exploiting weak passwords, phishing credentials, and navigating privileged access is a hacker's easiest way to infiltrate your organization. How can you fight back? Done right, identity is your first line of defense and the backbone of your security strategy. Join this session to learn how to leverage Okta to stop Identity breaches.
+
+So, if you are attending RSA, we hope you join us for this important session. Also, please stop by our booth and say hello!
+
+
+We will feature two topics that may be of keen interest to you: Shared Signal Framework (SSF) and Universal Logout. We would like to provide you with a brief understanding of each and resources for further exploration.
+
+
+## Shared Signal Framework (SSF)
+Does your current security solution wait until it sees suspicious activity in a critical application before logging an attacker out or prompting a legitimate user to re-authenticate? Chances are, accessing an essential application wasn't the first step an attacker took after compromising a user's account. In a genuinely zero-trust infrastructure, applications share information about user behavior that paints a holistic picture of whether an account's activity looks suspicious. Then each service responds appropriately for its threat model. 
+
+The Shared Signals Framework (SSF) is a huge step in enabling formerly siloed applications to see the whole picture of whether a user's behavior looks normal or suspicious. SSF is developed by the OpenID Foundation, which is also the home of standards like OIDC, FAPI, and MODRNA. 
+
+Although any software can send and receive these signals, Okta is uniquely positioned to work with our integrators to make SSF easy for identity administrators to use. 
+
+To learn more about the Shared Signals Framework, check out the OpenID foundation's [SSF Working Group](https://openid.net/specs/openid-sharedsignals-framework-1_0.html) or the explainer site [sharedsignals.guide](https://sharedsignals.guide/). Learn about how Okta and our integrators are deploying some of the world's first SSF implementations on [our developer podcast](TODO FIXME LINK)
+
+> Abstract on SSF filled out by E. Dunham. Also please replace link to podcast.
```

Comment:
```suggestion
```

Reply:
the above text is the abstract that I wrote in the google doc last week; is there more needed?